### PR TITLE
Describe the line out setting in the mixer's help

### PR DESCRIPTION
**Pull request by kcgen**
This was intended to be part of PR #1597 as was shown in the PRs screenshot, but the code [didn't make it.](https://github.com/dosbox-staging/dosbox-staging/pull/1597#issuecomment-1127630875)

This adds those changes, exactly as previously shown:

Screenshot from 2022-05-16 16-16-02

### CODE REVIEW

**Review by johnnovak — APPROVED** [review]
Looking good 👍🏻

File: src/hardware/mixer.cpp [hunk 1]
```diff
@@ -1016,26 +1015,29 @@ class MIXER final : public Program {
...
-	        "Examples:\n"
-	        "  [color=green]mixer[reset]\n"
-	        "  [color=green]mixer[reset] [color=cyan]master[reset] [color=white]50[reset] [color=cyan]record[reset] [color=white]60[reset] /noshow\n"
-	        "  [color=green]mixer[reset] /listmidi");
+		        "Displays or changes the current sound mixer settings.\n"
+		        "\n"
+		        "Usage:\n"
+		        "  [color=green]mixer[reset] [color=cyan]CHANNEL[reset] [color=white]VOLUME[reset] [/noshow]\n"
+		        "  [color=green]mixer[reset] [color=cyan]CHANNEL[reset] [color=white]LINEOUT[reset] [/noshow]\n"
+		        "  [color=green]mixer[reset] [/listmidi][reset]\n"
```
Comment (shermp) [inline src/hardware/mixer.cpp:1023]: Extra `[reset]` after `[/listmidi]`

**Review by shermp — APPROVED** [review]
Looks good. One tiny little (non) issue.

File: src/hardware/mixer.cpp (hunk 1 above)
Comment (kcgen) [reply src/hardware/mixer.cpp]: Fixed.

**Pull request merged**